Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,49 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
+    :type message: str
+    :param level: log level of the message
+    :type level: ``pyDKB.common.types.logLevel`` member
+    :param *args: additional prefixes (will be output between log
+                  level prefix and message body)
+    :type *args: str
+    """
+    if not logLevel.hasMember(level):
```

Comment:
Does not work with incorrect members because `level` is supposed to be member of `logLevel`, which prompts its call in this check:

    >>> log('abc', logLevel.WARN)
    2019-08-26 12:26:47 (WARN) (main) abc
    >>> log('abc', logLevel.WARNN)
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    AttributeError: 'Type' object has no attribute 'WARNN'